src/app/components/iterations-modal/iterations-modal.component.html

```diff
@@ -49,14 +49,14 @@
       </div>
       <div *ngIf="modalType=='create' || modalType=='update' || modalType=='start' || modalType=='createChild'"
         class="form-group datepicker-container">
-        <div class="date-container padding-right-5">
+        <div class="date-container padding-right-5 dib">
```

Comment:
I would prefer to call the class(styles already exists) itself rather than adding new line of css